addons/binding/org.openhab.binding.yeelight/README.md

```diff
@@ -0,0 +1,79 @@
+# Yeelight Binding
+
+This binding integrates the [Yeelight Lighting Product](https://www.yeelight.com/).
+
+
+## Supported Things
+
+[Yeelight LED White Bulb](https://www.yeelight.com/zh_CN/product/wifi-led-w)
+
+[Yeelight LED Color Bulb](https://www.yeelight.com/zh_CN/product/wifi-led-c)
+
+[Yeelight LED Color Stripe](https://www.yeelight.com/zh_CN/product/pitaya)
+
+Project will be updated to support future product.
```

Comment:
remove this line - it isn't really helpful...

Reply:
You should specify the configuration parameters for the Things.

Reply:
Like e.g. here: https://docs.openhab.org/addons/bindings/denonmarantz/readme.html#thing-configuration

Reply:
Are there any parameters that can be set for this?

Reply:
Ok, saw your link from below now.